### ISSUE TITLE
The cloud data sources Q&A should be clarified.

### ISSUE THREAD

**Issue by sqlchick** [MicrosoftDocs/powerbi-docs#1054]
The comment about not needing a gateway for cloud data sources such as Azure SQLDB is correct--but incomplete thus misleading. If a cloud service like Azure SQLDB resides within a corporate virtual network, behind a firewall, or requires a VPN, then it would require a gateway. More and more companies are implementing virtual networks to enhance security so it would be helpful to update the Q&A with a reference to that to make people aware that it's not always so black and white. Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d7103a5b-7dd8-c9df-0302-f374292a0934
* Version Independent ID: 2d08eb1c-6e9e-fb1f-c305-b75ae4bf70fc
* Content: [On-premises data gateway FAQ - Power BI](https://docs.microsoft.com/en-us/power-bi/service-gateway-onprem-faq)
* Content Source: [powerbi-docs/service-gateway-onprem-faq.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/live/powerbi-docs/service-gateway-onprem-faq.md)
* Service: **powerbi**
* Sub-service: **powerbi-gateways**
* GitHub Login: @mgblythe
* Microsoft Alias: **mblythe**

**Comment by maggiesMSFT** [comment]
Hi, @sqlchick -- thanks for your suggestion! I've assigned it to the author.

**Comment by mgblythe** [comment]
Thanks @sqlchick - we're redoing the gateway docs and I'll make sure we clarify this Q&A.

@v-doklo - the new version of this article is better, but please take a look at this specific Q&A and make sure it doesn't say "No" for cloud sources.

**Comment by mgblythe** [comment]
We moved the FAQ as part of a doc rewrite: https://docs.microsoft.com/en-us/data-integration/gateway/service-gateway-onprem-faq. We clarified the point that you brought up: 

Question: Do I need a gateway for cloud data sources like Azure SQL Database?
Answer: No. In general, the service will be able to connect to that data source without a gateway. However, for on-premises data sources, or if the data sources reside behind a firewall, or requires a VPN, or are on virtual networks, a data gateway may be needed.

Thanks!
